ionic2/app/pages/about/about.page.scss

```diff
@@ -1,3 +1,7 @@
 .about {
 
 }
+
+ion-item a{
+  text-decoration: none;
```

Comment:
I just found out, that putting it in the scss file is affecting all the ion-items.